musicd/config.go

```diff
@@ -59,6 +59,7 @@ type DbConf struct {
 }
 
 type CommonConf struct {
+	Debug	  *bool  `validate:"required"`
```

Comment:
Function of how the Validator lib deals with the default value for booleans. It simply doesn't work to have that as a "bool" (i.e. not as a "*bool") because then it is impossible to separate "param is set to false" and "param is not set at all".

Reply:
TIL.